### ISSUE TITLE
Add option to manipulate timestamp of blocks

### ISSUE THREAD

**Issue by 3esmit** [ether-camp/ethereum-sandbox#72]
I'm working on a contract with Lock period. I'm having to change the constants from 25 days to 5 minutes to test. Would be interesting if we have a plugin or feature to manipulate timestamp of the block.

If that is not possible, a target time to execute transaction would be good too.

**Comment by asinyagin** [comment]
@3esmit yeah, that seems to be a good feature for Studio. We have such call for testing purposes in the sandbox https://github.com/ether-camp/ethereum-sandbox/blob/master/api/sandbox.js#L172
Here's how you can use it https://github.com/ether-camp/ethereum-testing-reference/blob/43ab75e4fd33788700df01d19a04ba724dc540f3/test/vault/vault-test.js#L70

**Comment by 3esmit** [comment]
Nice, but I don't get how can I execute this javascript in my current project config. But as I see, you can manipulate the timestamp.
Would be good that block always use real timestamp, unless defined manually, and have a input to set timestamp.

A option would be to set a hard timestamp, so there could be blocks with wrong timestamp order or even multiple blocks at the same timestamp (something impossible in a real ethereum network due consensus). 
Other option would be to forward the clock that generates the timestamp to a new position defined. 
Any would match my needs, and I don't know which aprooch is better for most use cases.

**Comment by asinyagin** [comment]
It's a test-script which uses our testing framework. Here's more info on how to run such tests https://github.com/ether-camp/ethereum-testing-reference/.
